draft-ietf-lake-edhoc.md

```diff
@@ -416,12 +511,14 @@ EDHOC allows the communication or negotiation of various protocol features durin
 
```

Comment:
(Adding it here because I can't comment on l.508 - Communication/Negotiation of Protocol Features ) 

Again this subsection does not seem to fit at the same level as the other "Protocol Elements". I would say this is either "Features" or an appendix, or maybe even as the first subsection of "Message Formatting and Processing"